src/main/java/de/zalando/zmon/dataservice/Application.java

```diff
@@ -13,6 +23,28 @@
 @EnableConfigurationProperties
 public class Application {
 
+    private static final String JAEGER = "jaeger";
```

Comment:
These should be moved to its own [Const class](http://www.javapractices.com/topic/TopicAction.do?Id=2)
As a side note, the OpenTracingConfig interface would be wrong because it's an anti-pattern (Effective Java)

Reply:
I guess the following should do:

package de.zalando.zmon.dataservice.config;

public class OpenTracingProvider {
    private static final String JAEGER = "jaeger";
    private static final String LIGHTSTEP = "lightstep";
    private static final String INSTANA = "instana";
}

Reply:
correction:
package de.zalando.zmon.dataservice.config;

public class OpenTracingProvider {
    public static final String JAEGER = "jaeger";
    public static final String LIGHTSTEP = "lightstep";
    public static final String INSTANA = "instana";
}